routes/infrastructure/config.py

```diff
@@ -0,0 +1,52 @@
+"""Settings for Cloudfront distribution - any environment variables starting with
+`VEDA_` will overwrite the values of variables in this file
+"""
+from typing import Optional
+
+from pydantic import BaseSettings, Field
+
+
+class vedaRouteSettings(BaseSettings):
+    """Veda Route settings"""
+
+    cloudfront: Optional[bool] = Field(
+        False,
+        description="Boolean if Cloudfront Distribution should be deployed",
+    )
+
+    # STAC S#3 browser bucket name
+    stac_browser_bucket: Optional[str] = Field(
+        "", description="STAC browser S3 bucket name"
+    )
+
+    # API Gateway URLs
+    ingest_url: Optional[str] = Field(
+        "",
+        description="URL of ingest API",
+    )
+
+    domain_hosted_zone_name: Optional[str] = Field(
+        None,
+        description="Domain name for the cloudfront distribution",
+    )
+
+    domain_hosted_zone_id: Optional[str] = Field(
+        None, description="Domain ID for the cloudfront distribution"
+    )
+
+    cert_arn: Optional[str] = Field(
+        None,
+        description="Certificate’s ARN",
+    )
+
+    using_mcp_acct: Optional[bool] = False
```

Comment:
I think  `using_mcp_acct` is not used? Also, not sure about the intent but possibly we could be inferring this info from whether or not `VEDA_PERMISSIONS_BOUNDARY_POLICY_NAME`  is configured?